src/app/components/SandboxCrash.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { QueryRenderer, graphql } from 'react-relay/compat';
+import Relay from 'react-relay/classic';
+
+const SandboxCrashComponent = ({ admin }) => {
+  // The following line is designed to crash on purpose, unsafe property access
+  const isAdmin = admin.is_admin.foo.bar;
+
+  return (
+    <div>
+      {isAdmin}
+    </div>
+  );
+};
+
+const SandboxCrash = () => (
+  <QueryRenderer
+    environment={Relay.Store}
+    query={graphql`
+      query SandboxCrashQuery {
+        me {
+          is_admin
+        }
+      }
+    `}
+    render={({ props }) => (
+      <SandboxCrashComponent admin={props?.me} />
+    )}
+  />
+);
+
+export default SandboxCrash;
```

Comment:
We could even get rid of the query renderer to avoid compiling and generating unnecessary relay/graphql files and have, say,

const foo = {};

and render foo.bar.baz

what do you think?

Reply:
Good point.